docs/install.md

```diff
@@ -21,14 +21,13 @@ curl https://install.timeplus.com/oss | sh
 Once the `proton` binary is available, you can run Timeplus Proton in different modes:
 
 - **Local Mode.** You run `proton local` to start it for fast processing on local and remote files using SQL without having to install a full server
-- **Config-less Mode.** You run `proton server` to start the server and put the config/logs/data in the current folder. Then use `proton client -h 127.0.0.1` in the other terminal to start the SQL client.
-- **Server Mode.** You run `sudo proton install` to install the server in predefined path and a default configuration file. Then you can run `sudo proton server -C/etc/proton-server/config.yaml` to start the server and use `proton client` in the other terminal to start the SQL client.
+- **Config-less Mode.** You run `proton server` to start the server and put the config/logs/data in the current folder `proton-data`. Then use `proton client` in the other terminal to start the SQL client.
+- **Server Mode.** You run `sudo proton install` to install the server in predefined path and a default configuration file. Then you can run `sudo proton server -C /etc/proton-server/config.yaml` to start the server and use `proton client` in the other terminal to start the SQL client.
```

Comment:
I do noticed -C/path/to/yaml file works, even it's a bit surprising. Can you share me the code for how this -C is handled?

Reply:
also surprised me. yes, I already tried, it works.
but I believe this is not straightforward, for me, I probably first copy paste and then manually type a blank.

below code is done by bash script, but I compared with clickhouse latest one, seems they also has this `feature???`
```
➜  ck ./clickhouse server -C../proton-enterprise/programs/server/config.yaml
Processing configuration file '../proton-enterprise/programs/server/config.yaml'.
Merging configuration file '../proton-enterprise/programs/server/config.d/access_control.xml'.
Poco::Exception. Code: 1000, e.code() = 0, Exception: Failed to merge config with '../proton-enterprise/programs/server/config.d/access_control.xml': Exception: Root element doesn't have the corresponding root element as the config file. It must be <clickhouse>, Stack trace (when copying this message, always include the lines below):
0. DB::ConfigProcessor::processConfig(bool*, zkutil::ZooKeeperNodeCache*, std::shared_ptr<Poco::Event> const&) @ 0x0000000013513b2a
1. DB::ConfigProcessor::loadConfig(bool) @ 0x0000000013513e54
2. BaseDaemon::initialize(Poco::Util::Application&) @ 0x000000000da7be40
3. DB::Server::initialize(Poco::Util::Application&) @ 0x000000000d955898
4. Poco::Util::Application::run() @ 0x0000000015bd187a
5. DB::Server::run() @ 0x000000000d955650
6. Poco::Util::ServerApplication::run(int, char**) @ 0x0000000015bda6c7
7. mainEntryClickHouseServer(int, char**) @ 0x000000000d9528a6
8. main @ 0x0000000007b7000e
9. ? @ 0x00007fecd1cc4083
10. _start @ 0x000000000625302e
 (version 24.8.1.1666 (official build))
➜  ck
```


https://github.com/timeplus-io/proton/blob/d5dc45fa9ac91e65a949b4041eab8ade2b1e17d5/programs/bash-completion/completions/proton-bootstrap#L150-L165